### DISCUSSION TITLE
Cannot make build on mac book air m3

### DISCUSSION THREAD

**Discussion by siddharthsabron** [apache/ozone#7235] [General]
<img width="1103" alt="Screenshot 2024-09-24 at 1 32 56 PM" src="https://github.com/user-attachments/assets/335d5d68-c097-4618-bae1-28e80c2797f2">
tried building probuf from source but still 

any suggestion or help

**Comment by ivandika3** [comment]
Hi @siddharth1729 , have you checked out this guide to build Ozone using MacBook M3 https://github.com/apache/ozone/discussions/6783?

cc: @siddhantsangwan

**Reply by siddharthsabron** [reply]
@ivandika3 Thankx a lot for this resource made build looking forward to contribute in project . 
Thankx again
